.gitignore

```diff
@@ -55,9 +55,11 @@ npm-debug.log
 .env*
 engflow.crt
 *.key
+!tools/crates/vendor/**/*.key
 privkey.pem
 *~
 CMakeLists.txt
+!tools/crates/vendor/**/CMakeLists.txt
```

Comment:
I believe these two ignores can be replaced with a single `!tools/crates/vendor/**` put at the end of the file.

Reply:
Right, but this would pull in a bunch of other files, too, that are not required for checksum calculations.